app/views/users/registrations/new.html.erb

```diff
@@ -28,7 +28,7 @@
         <%= f.submit "Sign up", class: "govuk-button" %>
       </div>
     <% end %>
+    <%= link_to "Sign in", new_session_path(resource_name), class: "govuk-link govuk-body" %>
```

Comment:
I think you just need the govuk-link style here: 
```suggestion
    <%= link_to "Sign in", new_session_path(resource_name), class: "govuk-link" %>
```

Reply:
Talked in-person - without the govuk-body style this link is tiny for some reason. Bears investigating at some point, perhaps, but fine with just chucking in the extra style here.